src/object_detection.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+height = 10000              # vertical dimension in millimeters
+width = 20000               # horizontal dimension in millimeters
+cell_dim = 20               # dimension of cells in millimeters (cells are squares)
+
+cell_col = np.ceil(height/cell_dim)
+cell_row = np.ceil(width/cell_dim)
+
+# Tunable parameters to determine if something is an object.
+col_diff = 20           # Expected max difference between two adjacent cells in a column.
+counter_reps = 3        # Number of reps required to dictate it is an object.
+prev_diff = 1           # Amount of error needed to exceed for previous to be flags
+same_obj_diff = 20      # maximum diff between horizontal cells to be considered the same object
+
+
+def object_detection(points):
+
+    cells = np.zeros((cell_row, cell_col))
+
+    for tup in points:
+        # creates list with x, y, and z coordinate
+        x, y, z = tup
+        x = x//cell_col
+        y = y//cell_row
+
+        # Dictating the z value for the cell. Currently only finds the minimum value of the cell.
+        if x >= 0 and x < cell_col and y >= 0 and y < cell_row:
```

Comment:
Just leaving a comment here, because we may have to change the boundaries for x if it can be negative (which i figure it can). We'll know after I run the tests on the lidar with the visualizer.